rules/avoid-deleting-records-by-flagging-them-as-isdeleted/rule.md

```diff
@@ -1,25 +1,23 @@
 ---
 type: rule
-archivedreason: 
-title: Data - Do you avoid deleting records by flagging them as IsDeleted (aka Soft Delete)?
-guid: 56fff82d-9ed7-4aee-ab22-68e4c61fe162
+title: Data - Do you avoid deleting records by flagging them as IsDeleted (aka
```

Comment:
Personally, I prefer "Data - Do you avoid deleting records by using Soft Delete (IsDeleted flag)?" but considering it's an old rule, it might be better to keep it as is for Google juice.